src/System/src/FloatingBaseSystemDynamics.cpp

```diff
@@ -14,6 +14,18 @@
 using namespace BipedalLocomotion::System;
 using namespace BipedalLocomotion::ParametersHandler;
 
+Eigen::Matrix4d setHomogeneusTransform(Eigen::Ref<const Eigen::Vector3d> position,
+                                       Eigen::Ref<const Eigen::Matrix3d> rotation)
+{
+    Eigen::Matrix4d transform;
+    transform.topLeftCorner<3,3>() = rotation;
+    transform.topRightCorner<3,1>() = position;
+    transform(3,3) = 1;
+    transform.bottomLeftCorner<1,3>().setZero();
+
+    return transform;
+}
```

Comment:
We had added this function in `Conversions` https://github.com/dic-iit/bipedal-locomotion-framework/blob/43c8b35a2659af9dad910999a08c35766a95e473/src/Conversions/include/BipedalLocomotion/Conversions/CommonConversions.h#L25.
Would you not like to use it ?

Reply:
Because I forgot!

Reply:
https://github.com/dic-iit/bipedal-locomotion-framework/pull/108/commits/3673a8801b78e9f304d3133a5ab66b97d4bfa838